### PR TITLE
malky/active profile

### PR DESCRIPTION
**Pull request by MalkySinowitz**
on tab opened, get the user's settings from the DB and save them in local storage

### CODE REVIEW

**Review by batiFinkelshtein — APPROVED** [review]
this changes looks good

File: extension-ui/background.js [hunk 1]
```diff
@@ -82,3 +82,21 @@ chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
     return true; // כדי להורות שהתגובה היא אסינכרונית
   }
 });
+
+chrome.runtime.onInstalled.addListener(() => {
+  const userId = 'm123';
```
Comment (e171271) [inline extension-ui/background.js:87]: userId shouldn't be hard-code

File: extension-ui/background.js [hunk 2]
```diff
@@ -82,3 +82,21 @@ chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
...
+
+  fetch(`http://localhost:3000/api/settings/${userId}`)
```
Comment (e171271) [inline extension-ui/background.js:89]: move the basic url to variable

**Review by e171271 — CHANGES_REQUESTED** [review]

**Review by batiFinkelshtein — APPROVED** [review]
fix!

**Review by batiFinkelshtein — APPROVED** [review]

**Review by e171271 — APPROVED** [review]

**Review by e171271 — APPROVED** [review]

**Pull request merged**